source/lua/Mine.lua

```diff
@@ -88,7 +88,10 @@ local function SineFalloff(distanceFraction)
 end
 
 function Mine:Detonate()
-    if not self.active then return end
+    if not self.active then

```

Comment:
The ponit was for the mine to invulnerable during arming time, right? I fixed this another way.

Reply:
No the point was for the mine to not detonate during the arming time ;)